packages/tokens/dist/cloud-email/tokens.scss

```diff
@@ -1,6 +1,10 @@
 
 // Do not edit directly, this file was auto-generated.
 
+$token-border-radius-x-small: 3px;
```

Comment:
@jorytindall is this resultant naming ok? (With the dash in "x-small")

Reply:
Yep, looks good, that's how they're defined in Figma

Reply:
see comment above about naming. I have feelings that we should reconsider it.